sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADConditionalAccessResponseErrorHandler.java

```diff
@@ -49,6 +49,8 @@ public void handleError(ClientHttpResponse response) throws IOException {
         if (oauth2Error == null) {
             oauth2Error = this.oauth2ErrorConverter.read(OAuth2Error.class, response);
         }
+        //Handle conditional access policy, step 1.
+        //About conditional access: https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/howto-conditional-access-policy-all-users-mfa
```

Comment:
1. It's better to add one space after `//`.
2. Here, `/* */` is better than `//`.
3. We can break line if link is too long.